src/game/ticTacToeGame.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+exports.__esModule = true;
+exports.TicTacToe = void 0;
+var events_1 = require("../events/events");
+var types_1 = require("../events/types");
+var gameTypes_1 = require("../shared/types/gameTypes");
+var playerTypes_1 = require("../shared/types/playerTypes");
```

Comment:
`require` is sort of old school. We normally use: 

`import type { playerTypes } from '../shared/types/playerTypes'`